app/api/groups/accept_join_requests.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
-// swagger:operation POST /groups/{parent_group_id}/requests/accept groups users groupRequestsAccept
+// swagger:operation POST /groups/{parent_group_id}/requests/accept groups users groupJoinRequestsAccept
```

Comment:
I'll have a conflict with master on this line, you should rebase.

Reply:
done